Objects/typeobject.c

```diff
@@ -3398,15 +3398,64 @@ get_bases_tuple(PyObject *bases_in, PyType_Spec *spec)
     return PyTuple_Pack(1, bases_in);
 }
 
+static inline int
+check_basicsize_includes_size_and_offsets(PyTypeObject* type) {
```

Comment:
It's the sound of the central PEP 7 scrutinizer... 🎵 🎵 🎵 🪕 

```suggestion
check_basicsize_includes_size_and_offsets(PyTypeObject *type)
{
```

Reply:
OTOH, this function is hard to bend to PEP 7 without reflowing almost every line...